src/Facade/Services/ConsignmentFacadeService.cs

```diff
@@ -239,5 +244,28 @@ public IResult<IEnumerable<Consignment>> GetByRequestResource(ConsignmentsReques
 
             return this.GetAll();
         }
+
+        public IResult<IEnumerable<Consignment>> GetByInvoiceNumber(int invoiceNumber)
+        {
+            var invoice = this.invoiceRepository.FindById(invoiceNumber);
+            if (invoice == null)
+            {
+                return new SuccessResult<IEnumerable<Consignment>>(new List<Consignment>());
```

Comment:
not found?

Reply:
was originally notfound but it caused problems in the front end with the loading spinner.  if it had been a direct lookup of saying consignments/123 that returned nothing then i'd be notfounding but its more like the query returns no resullts which makes the front end work better